wls-gui-wahllokalsystem/src/App.vue

```diff
@@ -112,6 +112,10 @@
           title="Wahlvorstand"
           :to="ROUTE_WAHLVORSTAND"
         />
+        <v-list-item
+          title="Störungen"
+          :to="{ name: ROUTE_VORFAELLEUNDVORKOMMNISSE }"
```

Comment:
```suggestion
          :to="ROUTE_VORFAELLEUNDVORKOMMNISSE"
```
der name sollte hier direkt übergeben werden können, so wie beim wahlvorstand drüber. dafür muss aber der pfad in `router.ts` (aktuell `"/stoerungen"`) und der name in `constants.ts` (aktuell `"vorfaelleundvorkommnisse"`) übereinstimmen. bitte für `stoerungen` oder `vorfaelleundvorkommnisse` entscheiden.

Reply:
resolved with 05606ccd2801b787cb7ecf071eb814972ca41239